modules/upgrade/partials/incompat-changes.adoc

```diff
@@ -4,6 +4,8 @@
 +
 NOTE: This does not pertain to the `redpanda.remote.` topic properties, such as `redpanda.remote.delete`. The remote properties are not reset to their defaults by the AlterConfigs API to avoid the possibility of unintentionally disabling Tiered Storage for a topic, which could cause significant operational issues for clusters designed to use Tiered Storage exclusively or with object storage as the only durable storage tier.
 
-* Starting in version 24.2, xref:manage:cluster-maintenance/manage-throughput.adoc#client-throughput-limits[client throughput limits] are compatible with the https://cwiki.apache.org/confluence/display/KAFKA/KIP-546%3A+Add+Client+Quota+APIs+to+the+Admin+Client[AlterClientQuotas and DescribeClientQuotas^] Kafka APIs. Redpanda determines client throughput limits on a per-broker basis. In earlier versions, client throughput quotas were applied from cluster configurations on a per-shard basis. 
+* Starting in version 24.2, xref:manage:cluster-maintenance/manage-throughput.adoc#client-throughput-limits[client throughput limits] are compatible with the https://cwiki.apache.org/confluence/display/KAFKA/KIP-546%3A+Add+Client+Quota+APIs+to+the+Admin+Client[AlterClientQuotas and DescribeClientQuotas^] Kafka APIs. Redpanda determines client throughput limits on a per-broker basis. In earlier versions, client throughput quotas were applied from cluster configurations on a per-shard basis.
+
+* Starting in version 24.2, xref:reference:properties/cluster-properties.adoc#transaction_max_timeout_ms[transaction_max_timeout_ms] defaults to 15 minutes, which limits the timeout a transactional client producer can set when initiating transactions. When performing upgrades to version 24.2, if you have clients with timeouts larger than `transaction_max_timeout_ms`, then new producer init requests will fail due to this limit enforcement. You must either reduce the client set transaction timeout to less than 15 minutes or increase the default `transaction_max_timeout_ms` on the service side.
```

Comment:
```suggestion
* Starting in version 24.2, xref:reference:properties/cluster-properties.adoc#transaction_max_timeout_ms[transaction_max_timeout_ms] defaults to 15 minutes, which limits the timeout a client producer can set when initiating transactions. When performing upgrades to version 24.2, if you have clients with timeouts larger than `transaction_max_timeout_ms`, then new producer initialization requests will fail due to this limit enforcement. Either reduce the client set transaction timeout to less than 15 minutes or increase the default `transaction_max_timeout_ms` on the service side.
```

Reply:
> which limits the timeout a client producer can set when initiating transactions

instead can we say "which limits the timeout a transactional producer can set during initialization"